### DISCUSSION TITLE
SCSS to minified CSS with liquid templating

### DISCUSSION THREAD

**Discussion by tomtrigenex** [postcss/postcss#1546] [Q&A]
Hi, Im hoping someone can point me in the right direction.
I'm trying to convert SCSS into minified CSS. Normally I'd use an online tool or similar however it contains a lot of liquid templating that id like to be respected.

Would postcss be suitable for this or do you know of any other that might help please ?
Many thanks for your time !
Regards,
Tom

example
`
.site-footer {
        {% if section.settings.bg_footer_color != '' %}
        background: {{ section.settings.bg_footer_color }};
        {% endif %}
.
.
@media (max-width: 767px) {
        .footer-top .col-footer {
            border-bottom: 1px solid {{  section.settings.bg_footer_color }}; 
        }
    }
    }
`

**Comment by ai** [comment]
Nope. PostCSS can't do it out of the box.

But you can write a custom parser if you really want to do it with PostCSS (but it will be better to use a chain of specific tools).

**Reply by tomtrigenex** [reply]
Thanks,
By specific tools you mean things like gulp ??
Sorry if its a silly question
I appreciate your help
Cheers
Tom

**Reply by ai** [reply]
I mean find CLI till for liquid templates
